src/components/VirtualizedTreeSelect.js

```diff
@@ -110,7 +110,7 @@ class VirtualizedTreeSelect extends Component {
     //Checks whether the array of items already contain an option with the same valueKey (ID)
     if (sortedArr.has(option)) {
       //Deep copy of option, needed to distinguish option for multiple subtrees
-      option = JSON.parse(JSON.stringify(option));
+      option = structuredClone(option);
```

Comment:
I see the `structuredClone` function as unresolved, there is no implementation of it. A missing import perhaps?

Reply:
Can you provide a stacktrace or something? It is working fine for me. The support is quite decent. https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

Reply:
My bad, didn't know the function.